src/main/java/uk/gov/justice/hmpps/prison/service/OffenderDatesService.java

```diff
@@ -45,7 +45,7 @@ public SentenceCalcDates updateOffenderKeyDates(Long bookingId, RequestToUpdateO
                 .offenderBooking(offenderBooking)
                 .reasonCode("UPDATE")
                 .calculationDate(calculationDate)
-                .comments("CRD calculation ID: " + requestToUpdateOffenderDates.getCalculationUuid())
+                .comments("The information shown was calculated using the Calculate Release Dates service. The calculation ID is: " + requestToUpdateOffenderDates.getCalculationUuid())
```

Comment:
The DB column has a 240 character limit so this will fit OK

Reply:
I was going to ask about this. Thanks for checking it 👍🏻